api/envoy/type/v3alpha/token_bucket.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package envoy.type.v3alpha;
+
+option java_package = "io.envoyproxy.envoy.type.v3alpha";
+option java_outer_classname = "TokenBucketProto";
+option java_multiple_files = true;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/versioning.proto";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Token bucket]
+
+// Configures a token bucket, typically used for rate limiting.
+message TokenBucket {
```

Comment:
FWIW, we have similar config here https://github.com/envoyproxy/envoy/blob/dfe687d49574ef7eb1bf84867bf571e805a2bf97/api/envoy/api/v3alpha/core/config_source.proto#L96. May be we should consolidate them later once this gets in?

Reply:
Yes agreed we need to reconcile this. Let me think about this a bit more and possibly open some cleanup issues.